.ci/.docker-images.yml

```diff
@@ -0,0 +1,389 @@
+    build_opts: "--build-arg PYTHON_IMAGE=python:3.10-rc"
+
+  - <<: *apm-agent-python-docker-images
+    tag: "pypy-3"
+    build_opts: "--build-arg PYTHON_IMAGE=pypy:3"
+
+  # APM Agent Node.js Docker Images
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "17"
+    build_opts: "--build-arg NODE_VERSION=17"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "16"
+    build_opts: "--build-arg NODE_VERSION=16"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "16.0"
+    build_opts: "--build-arg NODE_VERSION=16.0"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "14"
+    build_opts: "--build-arg NODE_VERSION=14"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "14.0"
+    build_opts: "--build-arg NODE_VERSION=14.0"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "12"
+    build_opts: "--build-arg NODE_VERSION=12"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "12.0"
+    build_opts: "--build-arg NODE_VERSION=12.0"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "10"
+    build_opts: "--build-arg NODE_VERSION=10"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "10.0"
+    build_opts: "--build-arg NODE_VERSION=10.0"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "8"
+    build_opts: "--build-arg NODE_VERSION=8"
+
+  - <<: *apm-agent-nodejs-docker-images
+    tag: "8.6"
+    build_opts: "--build-arg NODE_VERSION=8.6"
+
+  # APM Agent Ruby Docker images
+
+  - <<: *apm-agent-ruby-docker-images
+    name: "apm-agent-jruby"
+    working_directory: ".ci/docker/jruby"
+    build_script: "./run.sh --action build --registry ${REGISTRY}/${PREFIX}"
+    test_script: "./run.sh --action test --registry ${REGISTRY}/${PREFIX}"
+    push_script: "./run.sh --action push --registry ${REGISTRY}/${PREFIX}"
+
+  - <<: *apm-agent-ruby-docker-images
+    tag: "ruby-3.0"
+    build_opts: "--build-arg RUBY_IMAGE='ruby:3.0'"
+
+  - <<: *apm-agent-ruby-docker-images
+    tag: "ruby-2.7"
+    build_opts: "--build-arg RUBY_IMAGE='ruby:2.7'"
+
+  - <<: *apm-agent-ruby-docker-images
+    tag: "ruby-2.6"
+    build_opts: "--build-arg RUBY_IMAGE='ruby:2.6'"
+
+  - <<: *apm-agent-ruby-docker-images
+    tag: "ruby-2.5"
+    build_opts: "--build-arg RUBY_IMAGE='ruby:2.5'"
+
+  - <<: *apm-agent-ruby-docker-images
+    tag: "ruby-2.4"
+    build_opts: "--build-arg RUBY_IMAGE='ruby:2.4'"
+
+  - <<: *apm-agent-ruby-docker-images
+    tag: "jruby-9.2"
+    build_opts: "--build-arg RUBY_IMAGE='jruby:9.2'"
+
+  # APM Agent RUM Docker images
+
+  - <<: *apm-agent-rum-docker-images
+    name: "node-playwright"
+    tag: "12"
+    working_directory: ".ci/docker/node-playwright"
+    build_opts: "--build-arg NODEJS_VERSION='12'"
+
+## This is also failing in Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/node-puppeteer/
+#  - <<: *apm-agent-rum-docker-images
+#    name: "node-puppeteer"
+#    tag: "12"
+#    working_directory: ".ci/docker/node-puppeteer"
+#    build_opts: "--build-arg NODEJS_VERSION='12'"
```

Comment:
not used

Reply:
https://github.com/elastic/apm-pipeline-library/pull/1986/commits/acba55801309efb779538c3bfba237c8137523de

Reply:
I actually just found a usage for this image:
- https://github.com/search?q=org%3Aelastic+node-puppeteer&type=code